features/in_onderzoek.feature

```diff
@@ -166,188 +165,283 @@ Functionaliteit: in onderzoek
 
     Abstract Scenario: persoon heeft <veld> in onderzoek
```

Comment:
Moet boven regel 166 niet ook aangegeven worden dat het een proxy scenario is (dus @proxy toevoegen)?
Dat wordt bij de voorgaande scenario's i.i.g. wel gedaan.

Ik mis die aanduiding in de rest van dit bestand overigens bij veel meer (abstract) scenario's.

Reply:
Nee, dat hoeft niet. De @gba tag is om aan te geven dat een scenario specifiek is voor de GBA API. Een scenario zonder deze tag geldt voor zowel de proxy als de GBA API. In alle zoek-XXX.features zijn om die reden beide tags niet gebruikt

Reply:
Ok duidelijk.

Reply:
De door mij bedoelde scenario's in dit feature bestand beschouwen het traject vanaf de BRP tot na de proxy als een blackbox aangezien ze die als 1 stap beschrijven. Ze beschrijven dus eigenlijk niet de 2 stappen die er in werkelijkheid worden uitgevoerd, nl. de transformatie die plaatsvindt vanuit het BRP naar de GBA API response en de transformatie die door de proxy wordt uitgevoerd.
In feite kunnen de provider ontwikkelaars op basis van dit bestand toch niet afleiden hoe zij vanuit de BRP de GBA response moeten vervaardigen?

Reply:
> De door mij bedoelde scenario's in dit feature bestand beschouwen het traject vanaf de BRP tot na de proxy als een blackbox aangezien ze die als 1 stap beschrijven. Ze beschrijven dus eigenlijk niet de 2 stappen die er in werkelijkheid worden uitgevoerd, nl. de transformatie die plaatsvindt vanuit het BRP naar de GBA API response en de transformatie die door de proxy wordt uitgevoerd. In feite kunnen de provider ontwikkelaars op basis van dit bestand toch niet afleiden hoe zij vanuit de BRP de GBA response moeten vervaardigen?

Het is veel handiger om te beschrijven wat de gewenste functionaliteit is ipv te beschrijven hoe de gewenste functionaliteit moet worden geïmplementeerd. Dit leidt tot implementation bleed